tests/common/plugins/conditional_mark/tests_mark_conditions.yaml

```diff
@@ -1468,6 +1517,7 @@ qos/test_qos_sai.py::TestQosSai::testIPIPQosSaiDscpToPgMapping:
       - https://github.com/sonic-net/sonic-mgmt/issues/12906
       - "topo_type in ['m0', 'mx']"
       - "topo_name not in ['t0', 't0-64', 't0-116', 't0-35', 't0-56', 't0-standalone-32', 't0-standalone-64', 't0-standalone-128', 't0-standalone-256', 'dualtor-56', 'dualtor-120', 'dualtor', 't0-80', 't0-backend', 't1-lag', 't1-64-lag', 't1-56-lag', 't1-backend', 't2', 't2_2lc_36p-masic', 't2_2lc_min_ports-masic'] and asic_type not in ['mellanox']"
+      - "asic_type in ['vs']"
```

Comment:
Can skip test for "qos/test_qos_sai.py" instead of each qos case?

Reply:
As our principle of conditional mark, if we have longer matching, we will only adopt the conditions in the longest matching. This means, if we have both `qos/test_qos_sai.py` and  `qos/test_qos_sai.py::<test case>`, we will use the conditions in the second entry.